test/unit/api/core/devicestate/deviceState.purge.test.js

```diff
@@ -0,0 +1,46 @@
+var should = require('should');
+var validateDeviceState = require('../../validator/deviceStateValidator.js');
+
+describe('DeviceState', function() {
+
+  describe('purge', function() {
+    
+    it('should return 2 devicestates only', function (done) {
+     	
+        var state = {
+            devicetype : 1,
+            value: 12.1,
+            datetime: '2010-01-01 22:00:00'
+        };
+        
+        gladys.deviceState.createByIdentifier('THIS_IS_MY_IDENTIFIER', 'test', 'binary', state)
+        .then(function(result){
+           options = {
```

Comment:
Be careful, try to use always "var" when you create a variable ! Otherwise, it's kind of global, and it's a bad thing ^^